package-lock.json

```diff
@@ -687,6 +687,276 @@
 			"integrity": "sha512-XRsRjdf+j5ml+y/6GKHPZbrF/8p2Yga0JPtdqTIY2Xe5ohJPD9saDJJLPvp9+NSBprVvevdXZybnj2cv8OEd0A==",
 			"dev": true
 		},
+		"python-indent-parser": {
+			"version": "0.1.3",
+			"dependencies": {
```

Comment:
Do you know why all of these packages are listed as dependencies, when `python-indent-parser` itself has no non-development dependencies?

Reply:
As described in the commit I pushed, I should have added the `--production` flag when installing the python-indent-parser package.